cheating-response.md

```diff
@@ -36,14 +37,14 @@ Once something is reported to a staff member, immediately contact the League Tea
 - Rob Kleiman - rob@mlh.io
 - Jamie Liao - jamie.liao@mlh.io
 
-You can contact them on slack (where you can also find their cell numbers). The main objectives of the meeting with the League Team are to find out the following:
+The main objectives of the meeting with the League Team are to find out the following:
 
 - What happened?
 - Are we doing anything about it?
 - Who is doing those things?
 - When are they doing them?
 
-Once League Team have been contacted they will work with the staff member to loop in the event organizer if they are not already involved.
+Once League Team have been contacted they will work with the MLH rep to loop in the event organizer if they are not already involved.
```

Comment:
Don't want to be implied that it's on-site support. Generalize to generic cheating response team. Evaluate when/where want to look into the cheating incident (likely post-event)